test/e2e/resources/fleet-capi-test.yaml

```diff
@@ -0,0 +1,4896 @@
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: DockerCluster
+metadata:
+  name: test1
+  namespace: fleet-default
+spec:
+  failureDomains:
+    fd1:
+      controlPlane: true
+    fd2:
+      controlPlane: true
+    fd3:
+      controlPlane: true
+    fd4:
+      controlPlane: false
+    fd5:
+      controlPlane: false
+    fd6:
+      controlPlane: false
+    fd7:
+      controlPlane: false
+    fd8:
+      controlPlane: false
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  labels:
+    cni: test1-crs-0
+    cluster-api.cattle.io/rancher-auto-import: "true"
```

Comment:
can we set the label in the code instead? We want to test a case with a label on the namespace too, we can reuse this template.

Reply:
Yeah, makes sense, will update.

Reply:
For now I’ve only added fleet scenario, and it has labels included by default. We can look at it later.